webapp/packages/plugin-user-profile/src/UserProfileForm/UserInfoPart/UserProfileFormInfo.tsx

```diff
@@ -5,43 +5,49 @@
  * Licensed under the Apache License, Version 2.0.
  * you may not use this file except in compliance with the License.
  */
+import { toJS } from 'mobx';
 import { observer } from 'mobx-react-lite';
 
-import { ColoredContainer, Container, Group, GroupTitle, InputField, Loader, useAutoLoad, useTranslate } from '@cloudbeaver/core-blocks';
-import { type TabContainerPanelComponent, useTab, useTabState } from '@cloudbeaver/core-ui';
+import { UserInfoMetaParametersResource, UserInfoResource } from '@cloudbeaver/core-authentication';
+import { ColoredContainer, Container, Group, GroupTitle, InputField, Loader, useTranslate } from '@cloudbeaver/core-blocks';
+import { useService } from '@cloudbeaver/core-di';
+import { type TabContainerPanelComponent, useTab } from '@cloudbeaver/core-ui';
 
-import type { UserProfileFormProps } from '../UserProfileFormService.js';
+import type { IUserProfileFormInfoState } from './IUserProfileFormInfoState.js';
 import { UserActiveAuthMethods } from './UserActiveAuthMethods/UserActiveAuthMethods.js';
 import { UserProfileFormInfoMetaParameters } from './UserProfileFormInfoMetaParameters.js';
-import type { UserProfileFormInfoPart } from './UserProfileFormInfoPart.js';
 
-export const UserProfileFormInfo: TabContainerPanelComponent<UserProfileFormProps> = observer(function UserProfileFormInfo({ tabId }) {
+export const UserProfileFormInfo: TabContainerPanelComponent = observer(function UserProfileFormInfo({ tabId }) {
   const translate = useTranslate();
+  const userInfoResource = useService(UserInfoResource);
+  const userInfoMetaParametersResource = useService(UserInfoMetaParametersResource);
+  const disabled = userInfoResource.isLoading() || userInfoMetaParametersResource.isLoading();
   const tab = useTab(tabId);
-  const tabState = useTabState<UserProfileFormInfoPart>();
 
-  useAutoLoad(UserProfileFormInfo, tabState, tab.selected);
-
-  const disabled = tabState.isLoading();
+  const state: IUserProfileFormInfoState = {
+    userId: userInfoResource.data?.userId || '',
+    displayName: userInfoResource.data?.displayName || '',
+    authRole: userInfoResource.data?.authRole || '',
+    metaParameters: toJS(userInfoMetaParametersResource.data || {}),
+  };
```

Comment:
In case you need some data from resources, you need to use `useResource` hook instead (to be sure that data is synchronized with the backend and up to date